services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -212,16 +213,45 @@ async def update(
         return FolderDB.from_orm(row)
 
 
-async def delete(
+async def delete_with_all_children(
```

Comment:
```suggestion
async def delete_recursively(
```

Reply:
so this deletes recursively only folders, right?

Reply:
correct